app/models/incomeSourceDetails/IncomeSourceDetailsResponseModel.scala

```diff
@@ -30,6 +31,7 @@ case class IncomeSourceDetailsError(status: Int, reason: String) extends IncomeS
 object IncomeSourceDetailsModel {
 
   def applyWithFields(nino: String,
+                      mtdbsa: Option[String],
```

Comment:
according to the api spec, mtdbsa is not an optional field

Reply:
done fixing